SyntheticsRunTestsTask/task.json

```
@@ -109,6 +109,13 @@
       "label": "JUnit Report",
       "defaultValue": "",
       "helpMarkDown": "The filename for a JUnit report if you want to generate one."
+    },
+    {
+      "name": "pollingTimeout",
+      "type": "int",
+      "label": "Polling Timeout",
+      "defaultValue": "",
+      "helpMarkDown": "The duration (in milliseconds) after which the orb stops polling for test results."
```

Not sure about the naming, is it extension for azure devops?
```suggestion
      "helpMarkDown": "The duration (in milliseconds) after which the extension stops polling for test results."
```